.github/workflows/release.yaml

```diff
@@ -1,4 +1,5 @@
 on:
+  pull_request: {}
```

Comment:
yeah that's a good idea let me see what i can do. ideally i want to avoid creating a duplicate job. otherwise we need to keep these jobs in sync. maybe there is a way to set a job name conditionally 💭

Reply:
alright now it looks like this with pull requests:

<img width="1090" alt="Screenshot 2023-11-30 at 4 53 43 PM" src="https://github.com/cilium/cilium-cli/assets/25494577/31718cae-20b8-4fce-a3ed-ea6cab781935">

Reply:
Nice, thanks!